video/vaapi.c

```diff
@@ -21,28 +21,50 @@
 
 #include "vaapi.h"
 #include "common/common.h"
+#include "common/global.h"
 #include "common/msg.h"
 #include "osdep/threads.h"
 #include "mp_image.h"
 #include "img_format.h"
 #include "mp_image_pool.h"
 #include "options/m_config.h"
 
+#ifdef _WIN32
+#include "osdep/windows_utils.h"
+#include "out/d3d11/context.h"
+#include "out/gpu/d3d11_helpers.h"
+#endif
+
 #include <libavutil/hwcontext.h>
 #include <libavutil/hwcontext_vaapi.h>
 
+#ifdef _WIN32
+#define DEV_PATH_DEFAULT NULL
+#define DEV_PATH_VALIDATE mp_dxgi_validate_adapter
+#else
+#define DEV_PATH_DEFAULT "/dev/dri/renderD128"
+#define DEV_PATH_VALIDATE validate_path
+
+static int validate_path(struct mp_log *log,
+                         const struct m_option *opt,
+                         struct bstr name, const char **value)
+{
+    return *value ? 0 : M_OPT_INVALID;
```

Comment:
made a mistake here:
```suggestion
    return (*value && **value) ? 0 : M_OPT_INVALID;
```

Reply:
fixed